docs/how-to-handle-public-calendar.md

```diff
@@ -34,11 +25,13 @@ meetings can join the group.
 All SIG maintainers have permission to edit the Public OpenTelemetry calendar.
 To get access to the calendar, please join the Google Group opentelemetry-calendar-contributors@googlegroups.com by submitting a request [here](https://groups.google.com/g/opentelemetry-calendar-contributors).
 If your identity is not recognizable from the e-mail you are using to request joining the group, please
-request to be added to this Google Group by creating an issue in this repository.
+request to be added to this Google Group by creating an issue in this repository. To create a new meeting, you will need
+to either link one of the OpenTelemetry Zoom accounts to your public calendar, or you can use the credentials for the
+cncf-opentelemetry-governance@lists.cncf.io  Google account, which is already linked to one of the CNCF Zoom accounts.
 
 Please keep the membership of this group up to date and accurate.
 
 ## Zoom bombing prevention
-All meetings now require a passcode to join: _77777_. This passcode is embedded into the calendar meeting invites and meeting notes documents.
+All meetings are created by Zoom with randomized passcodes, which are embedded into the shared calendar links.
 All members of opentelemetry-calendar-contributors@googlegroups.com have access to [this document](https://docs.google.com/document/d/1gt9ctxKGPrM_XTINqLgkSxYypdrczHkt2znjwgBU4UU/edit#)
 listing the host keys for our meetings and explaining how to deal with inappropriate behavior in Zoom.
```

Comment:
Will these still work? We didn't have any Zoom bombings recently but people who joined and accidentally had their mic open while they left their desk with a lot of noise at home or in the office, so we had to mute them using the host key.

Reply:
Yes, the host accounts remain the same. The fundamental difference is that we're using scheduled meetings (from the same host) rather than Zoom rooms.